tests/cosmogony_test.rs

```diff
@@ -224,6 +224,7 @@ fn test_lux_zone_types() {
     assert_eq!(lux.osm_id, "relation:407489");
     assert_eq!(lux.admin_level, Some(8));
     assert_eq!(lux.label, "Luxembourg, Canton Luxembourg, Lëtzebuerg");
+    assert_eq!(lux.country_code, Some("lu".into()));
```

Comment:
As @antoine-de suggested, it would be nice to test that the `country_code` is populated on the country itself (below line 256)

Reply:
Thank you both for catching that 👍🏼